guides/predicted-latency-based-scheduling/README.md

```diff
@@ -0,0 +1,295 @@
+# Feature: Predicted Latency based Load Balancing
+
+## Overview
+
+This feature introduces **predicted latency based load balancing**, where scheduling decisions are guided by real-time predictions of request latency rather than only utilization metrics like queue depth or KV-cache utilization.
```

Comment:
pls make it clear that the feature is experimental, both in the title above and here in the intro